calico-enterprise_versioned_docs/version-3.19-2/release-notes/index.mdx

```diff
@@ -148,3 +148,15 @@ This release is supported for use in production.
 * Security updates
 
 To update an existing installation of Calico Enterprise 3.19, see [Install a patch release](../getting-started/manifest-archive.mdx).
+
+### Calico Enterprise 3.19.1 operator-only bug fix release
+
+July 24, 2024
+
+Calico Enterprise 3.19.1 is now available with an update to the Tigera Operator.
+The Tigera Operator version has been updated to version 1.34.2.
+No other components have been changed.
+
+#### Bug fixes
+
+* Bug fixed!
```

Comment:
@rene-dekker I wasn't sure whether it's the same bug from 3.18. Can you supply the copy here or send it to me or Dan?

Reply:
```
- Starting with k8s v1.29, AKS applies an [ImageSet](../reference/installation/api#operator.tigera.io/v1.ImageSet) to the cluster. As a result, users wishing to upgrade Calico to enterprise may get stuck unknowingly, because the operator is checking for an ImageSet matching the target release version. This patch disables the check for an ImageSet, but only for upgrade scenarios where `Installation.Spec.Variant` changes.
- Removed a mutual dependency between logstorage and other components that could result in a degraded TigeraStatus if certificates are missing required key usages.
```

Reply:
I've added this, with edits. Thanks @rene-dekker